addons/editor-dark-mode/addon.json

```diff
@@ -1162,6 +1188,32 @@
       "name": "Scratch's default colors",
       "id": "scratch",
       "description": "The colors normally used by Scratch",
+      "values": {
+        "page": "#e5f0ff",
+        "primary": "#855cd6",
```

Comment:
The definition of the `primary-variant` CSS variable will need to be changed to correctly generate a darker version of this new color. Specifically, `r`, `g` and `b` in the `multiply` part should all be `0.85`.